sct-commons/pom.xml

```diff
@@ -161,23 +161,41 @@
                 <version>3.1.0</version>
                 <executions>
                     <execution>
-                        <id>xjc</id>
+                        <id>scl</id>
                         <goals>
                             <goal>xjc</goal>
                         </goals>
+                        <configuration>
+                            <sources>
+                                <source>${project.build.outputDirectory}/xsd/SCL2007B4/SCL.xsd</source>
+                                <source>${project.build.outputDirectory}/xsd/SCL_CoMPAS.xsd</source>
+                            </sources>
+                            <xjbSources>
+                                <xjbSource>${project.basedir}/src/main/resources/binding_configuration.xjb</xjbSource>
+                            </xjbSources>
+                            <packageName>org.lfenergy.compas.scl2007b4.model</packageName>
+                            <noPackageLevelAnnotations>true</noPackageLevelAnnotations>
+                            <clearOutputDir>false</clearOutputDir>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>epf</id>
+                        <goals>
+                            <goal>xjc</goal>
+                        </goals>
+                        <configuration>
+                            <sources>
+                                <source>${project.basedir}/src/main/resources/xsd/EPF.xsd</source>
+                            </sources>
+                            <xjbSources>
+                                <xjbSource>${project.basedir}/src/main/resources/binding_configuration.xjb</xjbSource>
```

Comment:
pourquoi ces paramètres de configurations sont-ils utilisés ici ? ça fonctionnait sans de mon côté je crois.

Reply:
This conf, help as try out `IsSetter` methods
```
<jaxb:globalBindings generateIsSetMethod="true"/>
```